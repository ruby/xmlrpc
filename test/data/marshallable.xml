<?xml version="1.0" ?>
<methodResponse>
  <params>
    <param>
      <value>
        <struct>
          <member>
            <name>___class___</name>
            <value>
              <string>Gem::Requirement</string>
            </value>
          </member>
        </struct>
      </value>
    </param>
  </params>
</methodResponse>