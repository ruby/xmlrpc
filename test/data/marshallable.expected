---
- true
- ___class___: Gem::Requirement
